portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,27 @@
     <link rel="stylesheet" href="style.css">
```

Comment:
not sure why i can't attach this comment to line 2, but..

Consider adding a lang="en" in the html tag: https://www.w3.org/International/questions/qa-html-language-declarations

This is important for assistive technology to know the language of the content.